src/neo-express/Node/NodeUtility.cs

```diff
@@ -20,8 +23,123 @@ namespace NeoExpress.Node
 {
     internal static class NodeUtility
     {
-        public static Task RunAsync(Store store, ExpressConsensusNode node, TextWriter writer, CancellationToken cancellationToken)
+        private static ulong GetNonce()
         {
+            Span<byte> nonce = stackalloc byte[sizeof(ulong)];
+            Random rand = new Random();
```

Comment:
a sequence of nonces might not be as random as you'd expect if this gets called in a loop (can make rand a static variable to avoid this problem).

Reply:
Good catch. I fixed a similar issue for [Neo-FX master branch](https://github.com/neo-project/neo-fx/blob/master/src/NeoFx/StaticRandom.cs) but missed this one.